src/SignhostAPIClient.Tests/PostbackTests.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using Newtonsoft.Json;
+using Signhost.APIClient.Rest.DataObjects;
+using Xunit;
+
+namespace Signhost.APIClient.Rest.Tests
+{
+	public class PostbackTests
+	{
+		[Fact]
+		public void PostbackTransaction_should_get_serialized_correctly()
+		{
+			string json = RequestBodies.MockPostbackValid;
+			var postbackTransaction = JsonConvert.DeserializeObject<PostbackTransaction>(json);
+
+			postbackTransaction.Id.Should().Be("b10ae331-af78-4e79-a39e-5b64693b6b68");
+			postbackTransaction.Status.Should().Be(TransactionStatus.InProgress);
+			postbackTransaction.Seal.Should().BeTrue();
+			postbackTransaction.Reference.Should().Be("Contract #123");
+			postbackTransaction.PostbackUrl.Should().Be("https://example.com/postback.php");
+			postbackTransaction.SignRequestMode.Should().Be(2);
+			postbackTransaction.DaysToExpire.Should().Be(30);
+			postbackTransaction.SendEmailNotifications.Should().BeTrue();
+			postbackTransaction.CreatedDateTime.Should().Be(DateTimeOffset.Parse("2016-08-31T21:22:56.2467731+02:00"));
+			postbackTransaction.CancelledDateTime.Should().BeNull();
+			(postbackTransaction.Context is null).Should().BeTrue();
```

Comment:
Would
```csharp
postbackTransaction.Context.Should().BeNullI();
```
not work? Instead of using 2 different ways.

Reply:
Did this because Context is dynamic and thus any method could be called on it.